kge/config.py

```diff
@@ -619,6 +621,16 @@ def rename_value_re(key_regex, old_value, new_value):
                     renamed_keys.add(key)
         return renamed_keys
 
+    # 26.02.2020
+    rename_key(
+        "negative_sampling.implementation.spo",
+        "negative_sampling.implementation.triple",
+    )
+    rename_key(
+        "negative_sampling.implementation.sp_po",
+        "negative_sampling.implementation.batch",
+    )
+
```

Comment:
This does not work. You need to rename the value for key negative_sampling.implementation. Please adapt and test.

Reply:
Everything else looks great.

Reply:
fixed